### DISCUSSION TITLE
Having to launch patched app through ReVanced?

### DISCUSSION THREAD

**Discussion by shefulloflight** [ReVanced/revanced-discussions#1123] [Question] [answered]
Hi, I'm pretty new to ReVanced, so I have a question: If what we're doing here is patching and then installing an APK, then how come on both my phone and tablet the correct app only opens when launched through ReVanced itself. Isn't the patched app installed on the system? Why does running the app from outside ReVanced seem to launch the unpatched app? Is this by design?

**Comment by oSumAtrIX** [comment]
You are not opening the patched app. The patched app may have a different package name

**Reply by shefulloflight** [reply]
Sorry, but this makes no sense to me: My S22 Ultra has a preinstalled YouTube app. I can't really remove it, so I uninstall the updates, then get the APK version that ReVanced recommends, then patch it successfully, then run it from the launcher. No patch. I run it from ReVanced. Patched. 😕

**Reply by oSumAtrIX** [reply]
YouTube patches modify the package name, so it installs as a separate app. You don't have to touch the pre installed app at all

**Reply by LisoUseInAIKyrios** [reply, marked as answer]
You can disable the stock YT app, or if your technically savvy you can completely remove it using a computer and adb.

Include the `Custom branding` patch if you're still getting confused 

To open links from the web in the patched app you need to go into the patched app info, and set it as default and turn on all the links.

**Reply by shefulloflight** [reply]
@LisoUseInAIKyrios Ah, I see. That's what I basically ended up doing: I used Shizuku + wireless debugging + Canta to remove the stock app, and now the patched app runs from my launcher by default all the time. (For whatever reason, I wasn't aware of the **Set as default** functionality before, which makes total sense now.) Thank you for explaining the `Custom branding` patch as well—everything works! 😄
